types/serialization_test.go

```diff
@@ -62,10 +64,13 @@ func TestBlockSerializationRoundTrip(t *testing.T) {
 				Commit: Commit{
 					Signatures: []Signature{Signature([]byte{1, 1, 1}), Signature([]byte{2, 2, 2})},
 				},
-				Validators: ValidatorSet{
-					Validators: []Validator{
-						{PublicKey: []byte{3, 3, 3}},
-						{PublicKey: []byte{4, 4, 4}},
+				Validators: &tmtypes.ValidatorSet{
+					Validators: []*tmtypes.Validator{
+						{Address: pubKey.Address(), PubKey: pubKey},
+						{Address: pubKey.Address(), PubKey: pubKey},
```

Comment:
Can we use different public keys here to maintain consistency with previous test?